Packs/TrendMicroEmailSecurity/Integrations/TrendMicroEmailSecurityEventCollector/TrendMicroEmailSecurityEventCollector_description.md

```diff
@@ -0,0 +1,32 @@
+## Trend Micro Email Security Event Collector
+Use this integration to collect operation logs from Trend Micro Email Security.
+You can also use the ***trend-micro-get-events*** command to manually collect events.
+
+In order to use this integration, you need to enter your Trend Micro Email Security credentials in the relevant integration instance parameters.
+
+### For the service URL parameter, attention to the following table:
+
+The value of service URL varies according to your location:
+| **Location** | **Service Root URL** |
+| --- | --- |
+| North America, Latin America and Asia Pacific | api.tmes.trendmicro.com |
+| Europe, the Middle East and Africa | api.tmes.trendmicro.eu |
+| Australia and New Zealand | api.tmes-anz.trendmicro.com |
+| Japan | api.tmems-jp.trendmicro.com |
+| Singapore | api.tmes-sg.trendmicro.com |
+| India | api.tmes-in.trendmicro.com |
+
+
+### For API Key parameter, attention to the following:
+
+1. Go to Administration > Service Integration.
+2. On the API Access tab, click Add to generate a key.
+The API Key is the global unique identifier for your application to authenticate its access to Trend Micro Email Security. It must be used together with the administrator account that created it. A new API Key is enabled by default.
+
+For more information about API Key see [here](https://docs.trendmicro.com/en-us/enterprise/trend-micro-email-security-online-help/configuring-administ/service-integration/api-access/obtaining-an-api-key.aspx)
+
+
+
+### Max Fetch
+Note: During an integration fetch, there are three types of fetches available, with the maximum fetch parameter set to 1000, 1000 logs will be retrieved for each type, resulting in a total of 3000 logs.
```

Comment:
```suggestion
Note: During an integration fetch, there are three types of fetches available. Wen the maximum fetch parameter is set to 1000, 1000 logs will be retrieved for each type, resulting in a total of 3000 logs.
```
Regarding this wording:
"There are three types of fetches..."
I don't understand. Can you list the different types of "fetches"? Should it be ..."three types of events"...?

Reply:
Yes, these are three types of events